src/main/java/edu/ucsb/ucsbcslas/controllers/CourseController.java

```diff
@@ -246,5 +246,26 @@
       throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Malformed CSV", e);
     }
   }
+
```

Comment:
The code LGTM.

But there is one change I'd like to see; the description says: 

```
PR resolved #167. 
```

That's not really true.    What I'd like to suggest is that you create a new issue (with a new issue number) that describes the work done in this PR, which is far less than the work described in issue #167 .   Issue #167 needs to "stay alive" for someone to work on it.   The work here is good *progress* towards addressing issue #167, but it's really just a piece broken off of #167.     

If you do that, I'm happy to approve this and see it merged.

Reply:
My bad... you already did this.